scm/driver/github/content.go

```diff
@@ -8,9 +8,8 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
-	"time"
-
 	"github.com/drone/go-scm/scm"
+	"time"
```

Comment:
import ordering is incorrect.  please revert this change so that `time` is grouped with other standard library imports.